addons/web/models/models.py

```diff
@@ -352,7 +415,7 @@ def group_id_name(value):
                     for group in groups
                 }
             # retrieve all possible values, and return them with their label and counter
-            selection = self.fields_get([field_name])[field_name]
+            selection = self.fields_get([field_name])[field_name]['selection']
```

Comment:
Oh my.  Isn't this a bugfix?
Maybe should be fixed in an earlier version...

Reply:
It seems so. This is why I want test the two methods.